src/modules/dashboard/sagas/utils/safeHelpers.ts

```diff
@@ -133,11 +162,116 @@ export const getIdFromNFTDisplayName = (displayName: string) => {
   return chunks[chunks.length - 1].substring(1);
 };
 
-export const getRawTransactionData = (txs: SafeTransaction[]) =>
-  txs.filter((t) => t.transactionType === 'rawTransaction');
+export const getRawTransactionData = (
+  zodiacBridgeModule: Contract,
+  transaction: SafeTransaction,
+) =>
+  zodiacBridgeModule.interface.functions.executeTransaction.encode([
+    transaction.recipient.profile.walletAddress,
+    Number(transaction.rawAmount),
+    transaction.data,
+    0,
+  ]);
+
+export const getTransferNFTData = (
+  zodiacBridgeModule: Contract,
+  safe: ColonySafe,
+  transaction: SafeTransaction,
+) => {
+  const safeAddress = onLocalDevEnvironment
+    ? LOCAL_SAFE_ADDRESS
+    : safe.contractAddress;
+  const tokenId = onLocalDevEnvironment
+    ? LOCAL_TOKEN_ID
+    : Number(transaction.nftData.id);
+  const erc721Address = onLocalDevEnvironment
+    ? LOCAL_ERC721_ADDRESS
+    : transaction.nftData.address;
+
+  if (!safeAddress) {
+    throw new Error('LOCAL_SAFE_ADDRESS not set in .env.');
+  }
+
+  if (!erc721Address) {
+    throw new Error('LOCAL_ERC721_ADDRESS not set in .env.');
+  }
+
+  const erc721Contract = getErc721(safe, erc721Address);
+
+  // eslint-disable-next-line max-len
+  const safeTransferFromFn = erc721Contract.interface.functions.safeTransferFrom.encode(
+    [safeAddress, transaction.recipient.profile.walletAddress, tokenId],
+  );
+
+  return zodiacBridgeModule.interface.functions.executeTransaction.encode([
+    erc721Address,
+    0,
+    safeTransferFromFn,
+    0,
+  ]);
+};
+
+export const getTransferFundsData = async (
+  zodiacBridgeModule: Contract,
+  safe: ColonySafe,
+  transaction: SafeTransaction,
+) => {
+  const safeAddress = onLocalDevEnvironment
+    ? LOCAL_SAFE_ADDRESS
+    : safe.contractAddress;
+  const tokenAddress = onLocalDevEnvironment
+    ? LOCAL_SAFE_TOKEN_ADDRESS
+    : transaction.token.address;
+
+  if (!safeAddress) {
+    throw new Error('LOCAL_SAFE_ADDRESS not set in .env.');
+  }
+
+  if (!tokenAddress) {
+    throw new Error('LOCAL_SAFE_TOKEN_ADDRESS not set in .env.');
+  }
+
+  const isSafeNativeToken = tokenAddress === AddressZero;
+
+  const getAmount = (): number => {
+    if (isSafeNativeToken) {
+      return moveDecimal(transaction.amount, transaction.token.decimals);
+    }
+    return 0;
+  };
+  const getData = async () => {
+    if (isSafeNativeToken) {
+      return '0x';
+    }
+    const TokenInterface = await getTokenInterface(safe, tokenAddress);
+
+    const tansferAmount = moveDecimal(
```

Comment:
typo

Reply:
This is where I was referring to about manually entering a number below 100 to get it working.

Reply:
Yes, that's because we were only minting 100 tokens, but I've changed it so that we mint `100000000000000000000` instead which is equal to `100` when formatted